esx_service/vmdk_ops.py

```diff
@@ -1469,6 +1473,19 @@ def execRequestThread(client_socket, cartel, request):
             reply_string = {u'Error': "Failed to parse json '%s'." % request}
             send_vmci_reply(client_socket, reply_string)
         else:
+            logging.debug("execRequestThread: req=%s", req)
+            # If req from client does not include version number, set the version to "1" by default
+            version = req["version"] if "version" in req else "1"
+            logging.debug("execRequestThread: version=%s", version)
+            if version != SERVER_PROTOCOL_VERSION:
+                if not version or int(version) < int(SERVER_PROTOCOL_VERSION):
+                    reply_string = err("vSphere Docker Volume Service client version ({}) is older than server version ({}), "
```

Comment:
Also, SERVER_PROTOCOL_VERSION is int() already, isn't it?
I suggest setting client_protocol_version (L 1478) to int as well
`client_protocol_version = int(req["version"] if "version" in req else 1` and droping int() conversion in other places